### PR TITLE
Update setup-cloud-management-gateway.md

### PR DESCRIPTION
**Pull request by russrimm**
These two resource providers were not enabled on a new azure subscription I just set up and CMG provisioning failed. I had to register them in order for provisioning to succeed.

### Summarize the change in the pull request title

Describe your change, specifically *why* you think it's needed.

Fixes #Issue_Number (if necessary)

### CODE REVIEW

**Comment by PRMerger10** [comment]
@russrimm : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@russrimm : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/core/clients/manage/cmg/setup-cloud-management-gateway.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ Use the following checklist to make sure you have the necessary information and
 - Starting in version 1802, select the **Azure Resource Manager deployment**. For more information, see [Azure Resource Manager](/sccm/core/clients/manage/cmg/plan-cloud-management-gateway#azure-resource-manager). You need the following requirements for an Azure Resource Manager deployment of CMG:  
 
     - Integration with [Azure AD](/sccm/core/servers/deploy/configure/azure-services-wizard) for **Cloud Management**. Azure AD user discovery isn't required.  
+    
+    - The **Microsoft.ClassicCompute** resource provider must be registered within the Azure subscription. For more information, see [Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-manager-supported-services).
```
Comment (aczechowski) [inline sccm/core/clients/manage/cmg/setup-cloud-management-gateway.md]: We need URLs to be language neutral, please remove `/en-us` from the address

**Comment by PRMerger10** [comment]
@russrimm : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Pull request merged**